pyproject.toml

```
@@ -10,7 +10,8 @@ requires-python = ">=3.9"
 description = "TBD"
 readme = "README.md"
 authors = [
-  { name = "Julius Busecke", email = "julius@ldeo.columbia.edu" }
+  { name = "Julius Busecke", email = "julius@ldeo.columbia.edu" },
+  { name = "Adam Subel", email = "adam.subel@nyu.edu"}
```

```suggestion
  { name = "Adam Subel", email = "adam.subel@nyu.edu"},
  { name = "Julius Busecke", email = "julius@ldeo.columbia.edu" }
```

I think since you are contributing most of the code you should come first here!